joebvp/VPmeasure.py

```diff
@@ -30,7 +31,7 @@
 print os.path.abspath(os.path.dirname(__file__))
 Ui_MainWindow, QMainWindow = loadUiType(modpath+'/mainvpwindow.ui')
 
-c=cfg.c/1e5
+c= c.to('km/s').value
```

Comment:
Thank you!

Reply:
You are welcome. It would be good to define the rest of fundamental constants this way, but these need to be replaced in a couple of distinct places among the package. Will leave that for you to make sure things don't break.